src/tasks/deploy.mk

```diff
@@ -93,15 +93,16 @@ deplo%: ## deploy: deploy the app to heroku
 
 change-api:
 	curl \
-  		--header "Content-Type: application/json" \
-  		--header "x-api-key: $(CHANGE_API_KEY)" \
+		--header "Content-Type: application/json" \
+		--header "x-api-key: $(CHANGE_API_KEY)" \
 		--request POST \
-  		--data "{ \
+		--data "{ \
 			\"user\": { \
 				\"githubName\": \"$(CIRCLE_USERNAME)\" \
 			}, \
 			\"environment\": \"production\", \
-			\"systemCode\": \"$(CIRCLE_PROJECT_REPONAME)\", \
+			\"systemCode\": \"$(shell curl https://next-registry.ft.com/v2/ | jq ".[] | select(.repository == \"https://github.com/${CIRCLE_USERNAME}/${CIRCLE_PROJECT_REPONAME}\") | .code")\", \
```

Comment:
Would CircleCI necessarily have `jq` installed? Might need to add it to https://github.com/Financial-Times/next-ci-shared-helpers

Should also update the `README.md` (and release notes).

Reply:
Good, shout. It does appear to be installed but better safe than sorry.

Reply:
Opened https://github.com/Financial-Times/next-ci-shared-helpers/pull/19.